CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Changelog
+## [1.5.3] - 2024-02-28
+### New
+- `PrimeDataUtils` has been added as a separate module, independent of the primeSdk, responsible for fetching data from REST APIs
```

Comment:
Pls call it DataUtils and replace the existing DataUtils with the prime backend.

Reply:
You will be adding those in future versions of the SDK, no? Those using the endpoints you mentioned wont upgrade to your current version of prime sdk.